.gitignore

```diff
@@ -62,3 +62,6 @@ PMMain/Midas/MidasGUI/GeneratedFiles/
 
 
 
+PMMain/Midas/MidasGUI/MidasGUI.vcxproj.user
+PMMain/Midas/MidasGUI/MidasGUI.vcxproj.filters
+PMMain/Midas/MidasGUI/MidasGUI.vcxproj
```

Comment:
Not needed. Please remove this change. *.vcxproj.user is already in the file, and others shouldnt be added.